lib/gds_api/test_helpers/publishing_api_v2.rb

```diff
@@ -230,13 +230,50 @@ def publishing_api_does_not_have_item(content_id)
         stub_request(:get, url).to_return(status: 404, body: resource_not_found(content_id, "content item").to_json, headers: {})
       end
 
+      # Stubs a request to links endpoint
+      #
+      # Requires a hash as paramenter
+      #
+      # @param links [Hash]
+      #
+      # @example
+      #
+      #   publishing_api_has_links(
+      #     "content_id" => "a-content-id",
+      #     "links" => {
+      #       topics: ['topic-content-id'],
+      #       mainstream_browse_pages: ['mainstream-content-id'],
+      #       parent: ['a-parent-content-id'],
+      #     },
+      #   )
       def publishing_api_has_links(links)
         links = links.with_indifferent_access
         url = PUBLISHING_API_V2_ENDPOINT + "/links/" + links[:content_id]
         stub_request(:get, url).to_return(status: 200, body: links.to_json, headers: {})
       end
 
+      # Stubs a request to expanded links endpoint
+      #
+      # Requires a hash as parameter
+      #
+      # @param links [Hash]
+      #
+      # @example
+      #   publishing_api_has_expanded_links(
+      #     content_id: "collection-id",
+      #     expanded_links: {
+      #       documents: [
+      #         {
+      #           "content_id" => "hi",
+      #           "title" => "Hi",
+      #           "base_path" => "/path/hi",
+      #           "document_type" => "guidance"
+      #         },
+      #       ]
+      #     }
+      #   )
       def publishing_api_has_expanded_links(links)
+        links = links.with_indifferent_access
```

Comment:
We shouldn't use `with_indifferent_access` here, it's not certain this is used in a Rails app.